src/employees/interface/employee.interface.ts

```diff
@@ -1,49 +1,45 @@
 interface EmployeeI {
-  employeeNo: any;
+  employeeNo: string;
   firstName: string;
   lastName: string;
   middleName: string;
-  fsCode: string;
-  bioCode: string;
   position: string;
   rank: string;
-  division: string;
   department: string;
-  designation: string;
-  dateHired: string;
-  yearsInService: string;
+  location: string;
+  isActive: boolean;
+  userGroup: string;
+  reportsTo: string;
+  dateHired: Date;
   employmentStatus: string;
-  endOfProbationary: string;
-  contractEndDate: string;
+  endOfProbationary: Date;
+  contractEndDate: Date;
   gender: string;
-  birthDate: string;
-  age: string;
+  birthDate: Date;
   contactNumber: string;
   taxExemption: string;
-  email: string;
+  companyEmail: string;
+  personalEmail: string;
   backAccountNo: string;
   civilStatus: string;
-  NumberOfDependents: string;
+  religion: string;
+  NumberOfDependents: number;
   sss: string;
   philHealth: string;
-  pagIbig: string;
+  pagIbig?: string;
   tin: string;
   city: string;
   zipCode: string;
   region: string;
   address: string;
-  course: string;
-  educationalAttainment: string;
-  schoolAttended: string;
+  course?: string;
+  educationalAttainment?: string;
```

Comment:
1. Remove column **educationalAttainment**.
2. New columns:
- highestEducationalAttainment - with enums - check HRIS Database Structure List of Values sheet
- elementaryYrFrom
- elementaryYrTo
- elementarySchoolAndAddress
- elementaryHonors
- secondaryYrFrom
- secondaryYrTo
- secondarySchoolAndAddress
- secondaryHonors
- tertiaryYrFrom
- tertiaryYrTo
- tertiarySchoolAndAddress
- tertiaryDegree
- tertiaryHonors
- postGradYrFrom
- postGradYrTo
- postGradSchoolAndAddress
- postGradDegree
- postGradHonors
- othersYrFrom
- othersYrTo
- othersSchoolAndAddress
- othersDegree
- othersHonors